libs/langgraph/src/channels/base.ts

```diff
@@ -6,6 +6,10 @@ import {
 } from "@langchain/langgraph-checkpoint";
 import { EmptyChannelError } from "../errors.js";
 
+export function isBaseChannel(obj: unknown): obj is BaseChannel {
+  return obj != null && typeof (obj as BaseChannel).lc_graph_name === "string";
```

Comment:
I don't think we should do this as a rule (`Send` has this populated) - can we add some other field?